prime-router/docs/workflows/Terraform.md

```diff
@@ -0,0 +1,103 @@
+GitHub repositories are source control repositories using Git version control along with additional features provided by GitHub, like pull requests and forking. GitHub offers a rich set of code collaboration tools that can be used to streamline the software development process as well as the infrastructure as code development process. The same GitHub version control workflow of using GitHub projects, git repositories, pull requests, and other features applies to managing infrastructure as code (IaC) deployments with HashiCorp Terraform as it does to application code builds and deployments.
+
+**GitHub Action to run Terraform Plan**
+The Terraform project code is located within the \prime-reportstream\operations\app\terraform\modules folder of the GitHub repository inside the main branch. The GitHub Actions are lo
+cated inside  \prime-reportstream\.github\workflows\ example is used at various workflows in the folder and can be easily modified to handle any other configuration.
+
+The following shows a simple GitHub Action to run the Terraform init, and plan commands, then store the Terraform Plan using a commit to the GitHub repository:
+name: Terraform Plan
+<div style="background-color:rgba(0, 0, 0, 0.0470588); text-align:center; vertical-align: middle; padding:40px 0; margin-top:30px">
+
+# configure to run on merges to 'main' branchbranchon:
+  push:
+    branches: [ master ]
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v2
+
+    # be sure to authenticate with cloud provider
+    - name: Login to Microsoft Azure
+      uses: azure/login@v1
+      with:
+        creds: ${{ secrets.AZURE_CREDENTIALS }}
+
+    - name: Setup Terraform
+      uses: hashicorp/setup-terraform@v2
+
+    - name: Terraform Init
+      run: terraform init
+
+    # generate Terraform Plan and store in .tfplan file
+    - name: Terraform Plan
+      run: terraform plan -out:terraform.tfplan
+
+    # commit Terraform plan if running on 'main' branch
+    # commit is made to the 'terraform-plan' branch
+    - name: Commit Terraform Plan for Approval
+      if: github.ref == 'refs/heads/main'
+      uses: peter-evans/create-pull-request@v4
+      with:
+        commit-message: 'Terraform Plan'
+        branch: 'terraform-plan'
+        base: 'main'
+        token: ${{ secrets.GITHUB_TOKEN }}
+
+</div>
+The previous GitHub Action example runs the following steps:
+1.	GitHub Action runs on pushes to the master branch
```

Comment:
Would you please make this a numbered list? It shows as one line if you look at it as it is - https://github.com/CDCgov/prime-reportstream/blob/feature/9453-DocumentWorkflows/prime-router/docs/workflows/Terraform.md

Reply:
Recommended Formatting changes are applied